backend/infrahub/graphql/mutations/branch.py

```diff
@@ -51,10 +45,13 @@ class BranchCreateInput(InputObjectType):
 class BranchCreate(Mutation):
     class Arguments:
         data = BranchCreateInput(required=True)
+        # TODO: We might want to change `background_execution` to `wait_until_completion` to keep consistent
```

Comment:
Good point, not sure if anyone is using background execution today., at a minimum we should mark it as deprecated
Either way I think it would be good to add `wait_until_completion` too to have a consistent behavior moving forward

Reply:
I added `wait_until_completion` and a related PR for sdk `InfrahubBranch.create` method that currently exposes `background_execution` even though it has no effect: https://github.com/opsmill/infrahub-sdk-python/pull/142